scripts/coverage.js

```diff
@@ -10,7 +10,7 @@ var Mocha = require('mocha'),
     Instrumenter = require('istanbul').Instrumenter,
     Report = require('istanbul').Report,
     Collector = new require('istanbul').Collector,
-    sourcefiles = ['index.js', 'extensions.js', 'render.js'],
```

Comment:
This is just the list of files to produce the coverage file for. It looks like the lowlevel.js is a consumer of the coverage.